code/src/index.css

```diff
@@ -1,13 +1,127 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Roboto Mono', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h2 {
+  font-family: 'Nunito', sans-serif;
+  font-size: 16px;
 }
+
+.outer-wrapper {
+  width: 100%;
+  display: flex;
+  align-items:center;
+  justify-content: center;
+}
+
+.inner-wrapper {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  margin-top: 20vh;
+  margin-bottom: 30vh;
+  position: relative;
+  min-width: 50%;
+}
+
+
+/* Styling for SendMessage-component */
+.outer-send-message-container {
+  border: solid 1px black;
+  box-shadow: 6px 6px 2px 1px black;
+  background-color: #f2f0f0;
+}
+
+.inner-send-message-container {
+  padding: 10px 20px 15px 20px;
+}
+
+.send-btn {
+  border: none;
+  border-radius: 20px;
+  padding: 10px 15px 10px 15px;
+  background-color: #ffaead;
+  margin-top: 5px;
+}
+
+.send-btn:hover {
+  background-color: white;
+}
+
+.text-area {
+  width: 90%;
+  padding: 10px;
+  border: solid 1px lightgray;
+}
+
+.red-font {
+  color: red;
+  font-size: 13px;
+}
+
+.normal-font {
+  color: black;
+  font-size: 13px;
+}
+
+/* Styling for ShowMessage-component */
+.outer-show-message {
+  border: solid 2px black;
+  margin-top: 20px; 
+  margin-bottom: 20px;
+  box-shadow: 6px 6px 2px 1px black;
+}
+
+.inner-show-message {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.message {
+  margin-top: 0px;
+  margin-bottom: 10px;
+  overflow: auto;
+}
+
+.heart-and-counter-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.heart-btn-like {
+  border-radius: 50%;
+  padding: 14px;
+  border: none;
+  font-size: 17px;
+  background-color: #ffaead;
+}
+
+.heart-btn-no-like {
+  border-radius: 50%;
+  padding: 14px;
+  border: none;
+  font-size: 17px;
+}
+
+.heart-btn-like:hover, .heart-btn-no-like:hover {
```

Comment:
Maybe remove the hover for mobiles. It gets "stuck" so it stays red when you like something, until you touch somewhere else on the page then it turns pink.